features/geography/geography_landing_page.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Landing page for a version of geography
+
+User can decide whether this is the correct geography
+
+@geography @happy_path @smoke
+Scenario: View the geography landing page
+
+     Given I go to a geography landing page
+     Then I can see the 'Area Types' dimension
+     Then I can see the description for the 'Area Type id 0'
```

Comment:
It feels like testing for 3 specific area types (using specific `area-type-n` ID as a selector) is going to be quite fragile, for example if we change what we put there then the tests break. Is it more valuable to test that a description/title renders for each area type that is there using a loop? Considering we can't say definitely that there will always be three of them.